.github/workflows/build.yml

```diff
@@ -36,6 +36,17 @@ jobs:
             image: docker:latest
         permissions:
             packages: write
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - backend    : Cuda
```

Comment:
Let's add the compiler family, currently `gcc` for the `OpenMP` backend and `nvidia` for the `Cuda` backend. The tags for the images could then become `gcc-OpenMP` and `nvidia-Cuda`

Reply:
I'd modify the preset names like this too, i.e. `gcc-OpenMP` and `Nvidia-Cuda`.

Reply:
OK, let's go! :car:

Reply:
Done.